pgx-examples/numeric/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "numeric"
+version = "0.0.0"
+edition = "2021"
+rust-version = "1.58"
```

Comment:
I don't think this line should be here since we don't actually have a MSRV?
```suggestion
```

Reply:
Was copy-paste from another example.  I'll look at cleaning them all up in a different PR>